### PR TITLE
Adds MysqlGroupToggleGateway and all its tests.

### PR DESCRIPTION
**Pull request by vokoblin**
Updates composer.lock to get latest Labs version.

Conflicts:
    composer.lock

### CODE REVIEW

File: src/Toggle/MysqlGroupToggleGateway.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * MysqlGroupToggleGateway constructor.
+     * @param Connection $connection
+     */
+    public function __construct( $connection )
```
Comment (MeliaBurnley) [inline src/Toggle/MysqlGroupToggleGateway.php]: should be `__construct (Connection $connection)`

File: test/Toggle/MysqlGroupToggleGatewayTest.php [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    {
+        parent::setUp();
+
+        $connectionParams = array(
+            'dbname' => 'labs',
+            'user' => 'root',
+            'password' => '',
+            'host' => 'localhost',
+            'driver' => 'pdo_mysql',
+        );
```
Comment (danielburnley) [inline test/Toggle/MysqlGroupToggleGatewayTest.php]: You probably don't want to commit your database config :|

File: test/Toggle/MysqlGroupToggleGatewayTest.php [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    /**
+     * @test
+     */
+    public function givenExistentGroupTogglesFound_MysqlGroupToggleGateway_ReturnsArrayOfGroupToggles()
+    {
+        $releaseName = 'Test group toggle 1';
+        $url = 'a helpful url';
+        $id = $this->addRelease( $releaseName, $url );
```
Comment (MeliaBurnley) [inline test/Toggle/MysqlGroupToggleGatewayTest.php]: Any reason they're not inline? 

`$this->addRelease('Test group toggle 1', 'a helpful url')`

File: test/Toggle/MysqlGroupToggleGatewayTest.php [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    /**
+     * @test
+     */
+    public function givenExistentGroupTogglesAndNonGroupTogglesFound_MysqlGroupToggleGateway_ReturnsArrayOfGroupTogglesOnly()
+    {
+        $releaseName = 'Test group toggle 2';
+        $url = 'a helpful url';
+        $id = $this->addRelease( $releaseName, $url );
```
Comment (MeliaBurnley) [inline test/Toggle/MysqlGroupToggleGatewayTest.php]: same as above

File: test/Toggle/MysqlGroupToggleGatewayTest.php [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+        $expectedToggles = [ $expectedToggle, $expectedToggle2 ];
+        $returnedToggles = $this->gateway->getAllGroupToggles();
+
+        $this->assertEquals( $expectedToggles, $returnedToggles );
+    }
+
+    /**
+     * @throws \Doctrine\DBAL\Exception\InvalidArgumentException
+     */
+    private function deleteAddedReleases()
```
Comment (MeliaBurnley) [inline test/Toggle/MysqlGroupToggleGatewayTest.php]: all these should be before the setUp

**Pull request merged**